.ci/tav.json

```diff
@@ -23,6 +23,7 @@
     "memcached",
     "mongodb",
     "mongodb-core",
+    "mongoose",
```

Comment:
THis puts us at 34 * 7 = 238 combinations, so still under the 256 GH actions jobs limit. All good.